v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/migrations/shard/Shard.java

```diff
@@ -133,6 +136,14 @@ public Map<String, String> getDbNameToLogicalShardIdMap() {
     return dbNameToLogicalShardIdMap;
   }
 
+  public Boolean getIsShardedMigration() {
```

Comment:
Context:
We are changing the `Shard` class which is shared between tempaltes.
What are the implications of setting this to true or false in the context of bulk?
If this is flag is entirely ignored by any template, could we document this in the readme as well as the shard.java file as a commendtr?